examples/features/broker-connection/amqp-sending-overssl/src/main/java/org/apache/activemq/artemis/jms/example/BrokerConnectionSenderSSL.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.activemq.artemis.jms.example;
+
+import javax.jms.Connection;
+import javax.jms.ConnectionFactory;
+import javax.jms.MessageConsumer;
+import javax.jms.MessageProducer;
+import javax.jms.Queue;
+import javax.jms.Session;
+import javax.jms.TextMessage;
+
+import org.apache.qpid.jms.JmsConnectionFactory;
+
+/**
+ * This example is demonstrating how messages are transferred from one broker towards another broker
+ * through the sender element on a AMQP Broker Connection.
+ */
+public class BrokerConnectionSenderSSL {
+
+   public static void main(final String[] args) throws Exception {
+      Connection connectionOnServer0 = null;
+      ConnectionFactory connectionFactoryServer0 = new JmsConnectionFactory("amqp://localhost:5671");
```

Comment:
I know the target of the example is SSL between the brokers...but it might make more sense to use SSL for the clients too. Then you would only need one acceptor.

This basically makes it seem like SSL isnt being used.

Reply:
I'm not showing how to use SSL on  the client here.. .this is already covered elsewhere.

Reply:
I know that, as I commented. Its just weird though. The example broker config is actually more complicated due to not using SSL for the client...which wouldnt add any complexity since all the stores are already in place.

Reply:
I would not do that on this version.. there's a lot of stuff to cover already.

Reply:
I'm sending a change for this on the PR #3318